src/libs/ReportUtils.js

```diff
@@ -1575,6 +1580,11 @@ function getModifiedExpenseOriginalMessage(oldTransaction, transactionChanges, i
         originalMessage.currency = lodashGet(transactionChanges, 'currency', originalMessage.oldCurrency);
     }
 
+    if (_.has(transactionChanges, 'category')) {
+        originalMessage.oldCategory = TransactionUtils.getCategory(oldTransaction);
+        originalMessage.newCategory = transactionChanges.category;
```

Comment:
To match with old dot schema https://github.com/Expensify/Web-Expensify/blob/e8b170b19d63534d0d8679cd9e253a2130e17455/lib/Transaction.php#L725-L727 we need to make this `category`
```suggestion
        originalMessage.category = transactionChanges.category;
```

Reply:
Oops yeah, that will also match Auth too. Sorry jetlagged 😓